terraform/testcases/configmap_provider_https/otconfig.tpl

```diff
@@ -0,0 +1,37 @@
+extensions:
+  pprof:
+    endpoint: 0.0.0.0:1777
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:${grpc_port}
+
+processors:
+  batch:
+  metricstransform:
+    transforms:
+      - include: ".*${testing_id}"
+        match_type: regexp
+        action: update
+        operations:
+          - action: add_label
+            new_label: "testingId"
+            new_value: "${testing_id}"
+
+exporters:
+  logging:
+    loglevel: debug
```

Comment:
should we use loglevel:debug here (and in other configs) or as below:
```
exporters:
  logging:
    verbosity:
```

Reply:
I will do this in a different pr because I want to fix all occurrences.